examples/first-frame-network/main.cpp

```diff
@@ -104,14 +105,29 @@ int main(int argc, char *argv[]) {
         return 0;
     }
     aditof::Frame frame;
-
-    status = camera->requestFrame(&frame);
-    if (status != Status::OK) {
-        LOG(ERROR) << "Could not request frame!";
-        return 0;
-    } else {
-        LOG(INFO) << "succesfully requested frame!";
+    
+    clock_t current_ticks, delta_ticks;
+    clock_t fps = 0;
+
+    while (true)
+    {
+        current_ticks = clock();
+        status = camera->requestFrame(&frame);
+
+        delta_ticks = clock() - current_ticks; //the time, in ms, that took to render the scene
+        if(delta_ticks > 0)
+            fps = CLOCKS_PER_SEC / delta_ticks;
+        std::cout << fps << std::endl;
```

Comment:
This is a good way to measure the duration of requestFrame() but should not be part of the first-frame-network example.

Reply:
Removed